hbw/app/javascript/packs/hbw/components/form.js.jsx

```diff
@@ -156,6 +158,15 @@ modulejs.define('HBWForm', ['React', 'jQuery', 'HBWError', 'HBWFormDatetime',
       return null;
     };
 
+    deleteControl = () => {
+      if (!this.props.form.hide_delete_button) {
+        return <Delete env={this.props.env}
+                       processInstanceId={this.props.processInstanceId} />;
```

Comment:
Let's prefetch needed props first of all with `const { form, env, processInstanceId } = this.props`